core_java/grep/src/main/java/ca/jrvs/apps/practice/HelloWorld.java

```diff
@@ -1,10 +1,60 @@
 package ca.jrvs.apps.practice;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 class HelloWorld {
 
   // Your program begins with a call to main().
   // Prints "Hello, World" to the terminal window.
-  public static void main(String args[]) {
+  public static void main(String args[]) throws IOException {
     System.out.println("Hello, World");
+    int[][] test = {{1, 7}, {1, 2}};
+//    File[] listedFiles = new File(
+//        "/home/centos/dev/jarvis_data_eng_akshay/core_java/grep/src/main/java").listFiles();
+//    Stream<Path> files = Files
+//        .walk(Paths.get("/home/centos/dev/jarvis_data_eng_akshay/core_java/grep/src/main/java"));
```

Comment:
Did you paste these comments here on purpose?

Reply:
Good catch

Reply:
But the hello world stuff is just for testing